packages/host/app/components/operator-mode/code-mode.gts

```diff
@@ -166,7 +170,7 @@ export default class CodeMode extends Component<Signature> {
   @tracked private card: CardDef | undefined;
   @tracked private cardError: Error | undefined;
   @tracked private userHasDismissedURLError = false;
-  @tracked private selectedElement: Element | undefined;
+  @tracked private _selectedElement: Element | undefined;
```

Comment:
if it's `private` there is no need to use `_` prefix to indicate that this is not part of the public API

Reply:
if you really wanna make it hands off, then use `#selectedElement`

Reply:
I meant to actually rename the selectedElement since im running out of name and I didn't want to use something like `selectedElementInFile`